.gitignore

```diff
@@ -1,4 +1,10 @@
 /cloudinit_script_encrypted.txt
+
+# Conveniences maintained locally by developers
```

Comment:
Can we put some of the comments from the commit in the file itself instead of this comment? I think most people know .venv but it wouldn't hurt to mention what it was. And then one comment for both of the vscode things to show they are together with vscode?

Reply:
done